packages/fxa-settings/src/components/PageAvatar/en-US.ftl

```diff
@@ -20,8 +20,8 @@ avatar-page-rotate-button = Rotate
 avatar-page-camera-error = Could not initialize camera
 avatar-page-new-avatar =
   .alt = new profile picture
-avatar-page-file-upload-error = There was a problem uploading your profile picture
-avatar-page-delete-error = There was a problem deleting your avatar
+avatar-page-file-upload-error-2 = There was a problem uploading your profile picture.
+avatar-page-delete-error-2 = There was a problem deleting your avatar.
```

Comment:
Since we're making a new string we should also use consistent vocabulary. How about "profile picture"?

Reply:
Nice! :+1: